### PR TITLE
Option to synchronize simulation with real-time

### PR DESCRIPTION
**Pull request by chria**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -2,6 +2,8 @@
 --- PyFMI-2.9.7 ---
     * Added an argument to ResultDymolaBinary to allow for reading updated
       data from the loaded file.
+    * Added option to allow for synchronizing simulation with (scaled)
+      real-time.
```
Comment (modelonrobinandersson) [inline CHANGELOG:6]: Would it be more clear if we mention the name of the new option? Like "Added a new option 'synchronize_simulation' to ..."?

File: tests/test_fmi.py [hunk 2]
```diff
@@ -1155,6 +1155,71 @@ def test_simulation_without_initialization(self):
...
 
         nose.tools.assert_raises(FMUException, model.simulate, options=opts)
+    
+    @testattr(stddist = True)
+    def test_simulation_with_syncronization_exception_ME(self):
+        model = Dummy_FMUModelME2([], "bouncingBall.fmu", os.path.join(file_path, "files", "FMUs", "XML", "ME2.0"), _connect_dll=False)
+        opts = model.simulate_options()
+        opts["synchronize_simulation"] = "Hej"
+        
+        nose.tools.assert_raises(FMUException, model.simulate, options=opts)
```
Comment (modelonrobinandersson) [inline tests/test_fmi.py:1165]: I think it is better if we assert that it raises fmi.InvalidOptionException and not the one the exception extends. I also propose adding a docstring to the tests for maintainability.

File: src/pyfmi/fmi_algorithm_drivers.py [hunk 3]
```diff
@@ -159,6 +159,18 @@ class AssimuloFMIAlgOptions(OptionBase):
             example is filter = "*der" , stor all variables ending with
             'der'. Can also be a list.
             Default: None
+            
+        synchronize_simulation --
+            If set, the simulation will be syncronized to real-time or a
+            scaled real-time, if possible. The available options are:
+                True: Simulation is syncronized to real-time
```
Comment (modelonrobinandersson) [inline src/pyfmi/fmi_algorithm_drivers.py:166]: I think you accidentally dropped the letter h from the word syncronized

**Review by modelonrobinandersson — CHANGES_REQUESTED** [review]
Looks good I have nothing major to add. Just to change the exceptions we catch in the tests, add short docstring to the tests and fix the inconsistency between syncronized and synchronized

File: CHANGELOG (hunk 1 above)
Comment (chria) [reply CHANGELOG]: Thanks, fixed!

File: tests/test_fmi.py (hunk 2 above)
Comment (chria) [reply tests/test_fmi.py]: Agreed, fixed.

File: src/pyfmi/fmi_algorithm_drivers.py (hunk 3 above)
Comment (chria) [reply src/pyfmi/fmi_algorithm_drivers.py]: Right, fixed.

**Pull request merged**